NuRadioReco/detector/RNO_G/db_mongo_read.py

```diff
@@ -720,7 +735,38 @@ def get_channel_signal_chain_measurement(self, station_id=None, channel_id=None,
 
 
     def get_component_data(self, component_type, component_id, supplementary_info, primary_time, verbose=True, sparameter='S21'):
-        """ returns the current primary measurement of the component, reads in the component collection"""
+        """
+        returns the current primary measurement of the component, reads in the component collection
```

Comment:
Minor comment, I think with the PR #788 I am adding a wrapper around this function within the detector class which probably should be used by users.

Reply:
Then maybe you can update this docstring / the module docstring in #788? Feel free to copy and paste stuff from here if you think it applies.